db/migrate/20160118124600_create_remote_execution_features.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRemoteExecutionFeatures < ActiveRecord::Migration
+  def change
+    create_table :remote_execution_features do |t|
+      t.string :label, :index => true, :null => false
+      t.string :name
+      t.string :description
+      t.text :provided_inputs, :null => true
+      t.integer :job_template_id, :null => true
```

Comment:
do you want to be explicit about `null`? true is default so it's not required here and on line above